uk.co.compucorp.civicrm.hrleaveandabsences/CRM/HRLeaveAndAbsences/Page/AbsenceTab.php

```diff
@@ -6,6 +6,22 @@
 class CRM_HRLeaveAndAbsences_Page_AbsenceTab extends  CRM_Core_Page {
   public function run() {
     CRM_Utils_System::setTitle(ts('Absence'));
+
+    self::registerScripts();
     parent::run();
   }
+
+  static function registerScripts() {
```

Comment:
removed registerScripts as now it was only one line of change which is added to main function